quacc/recipes/qchem/core.py

```diff
@@ -0,0 +1,103 @@
+"""
+Core recipes for the Q-Chem
+"""
+from __future__ import annotations
+
+from copy import deepcopy
+from typing import Literal
+
+from ase.atoms import Atoms
+from monty.dev import requires
+
+from quacc.schemas.ase import (
+    summarize_opt_run,
+    summarize_run,
+    summarize_thermo_run,
+    summarize_vib_run,
+)
+from quacc.util.calc import run_ase_opt, run_ase_vib, run_calc
+from quacc.util.thermo import ideal_gas
+from quacc.calculators.qchem import QChem
+
+def static_job(
```

Comment:
The way Quacc is currently structured, I like to have some "default" values set that are reasonable. See https://github.com/Quantum-Accelerators/quacc/blob/ea1916e5929db032e0c5b0b3f92ed75c22b031c0/quacc/recipes/gaussian/core.py#L61-L77 for an example in the Gaussian calculator. I also like to have the user be forced to supply certain parameters as kwargs, such as the functional and basis set.